### PR TITLE
Unstructured scoring function

### PR DESCRIPTION
**Pull request by EhrenbruderRoschtbraten**
Implemented the unstructured scoring function.

### CODE REVIEW

**Review by 1Maxnet1 — APPROVED** [review]
Looks good to me and works as expected :+1:

**Review by JWG-1 — APPROVED** [review]
worked on my machine, test trained on epoch

File: model/models.py [hunk 1]
```diff
@@ -214,3 +214,38 @@ def forward(self, sub,rel):
...
+class CompGCN_Unstructured(CompGCNBase):
+	"""
+	Implements the Unstructured scoring function.
+
+	The Unstructured scoring function cannot take relationships into account, so it assumes, that head and tail entity
+	vectors are similar.
+	"""
+	def __init__(self, edge_index, edge_type, params=None):
+		"""
+		Init the ComGCN Base and all the layer used in the score function Unstructured.
```
Comment (RaphaW) [inline model/models.py]: *CompGCN

**Review by RaphaW — COMMENTED** [review]
Just a small typo. Otherwise it works for me. Thanks

**Review by A7exSchin — APPROVED** [review]

File: model/models.py (hunk 1 above)
Comment (EhrenbruderRoschtbraten) [reply model/models.py]: Done

**Pull request merged**